aws/table_aws_iam_policy.go

```diff
@@ -233,6 +240,28 @@ func getPolicyVersion(ctx context.Context, d *plugin.QueryData, h *plugin.Hydrat
 	return version, nil
 }
 
+func getPolicyUsage(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getPolicyUsage")
+	policy := h.Item.(*iam.Policy)
+
+	// Create Session
+	svc, err := IAMService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	params := &iam.ListEntitiesForPolicyInput{
+		PolicyArn: policy.Arn,
+	}
+
+	usage, err := svc.ListEntitiesForPolicy(params)
```

Comment:
@cjsrkd3321 could you please use API pagination -  ListEntitiesForPolicyPages

Reply:
modification completed.